### PR TITLE
Add Cherrypy

### PR DESCRIPTION
**Pull request by ThePeeps191**
## Adding a language

- [x] The code displays "Hello World"
- [x] I have updated the readme to include the new language
- [x] I have incremented the language count in the readme
- [x] I have no association with the language

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -254,7 +254,8 @@ Make sure to see [contributing.md](/contributing.md) for instructions on contrib
 * [Cheddar](c/Cheddar.cheddar)
 * [Cheese++](c/Cheese%2B%2B.cheese)
 * [Chef](c/Chef.ch)
-* [Chem](c/Chem)
+* [Chem](c/Chem
```
Comment (Richienb) [inline readme.md:257]:
```suggestion
* [Chem](c/Chem)
```

**Comment by Richienb** [comment]
It appears that the intended title for the filename should be **CherryPy**

**Comment by ThePeeps191** [comment]
yes, I will fix those problems right now

**Comment by ThePeeps191** [comment]
ill make another pull request later

**Pull request closed without merging**